umh.docs.umh.app/content/en/docs/getstarted/installation.md

```
@@ -63,7 +63,8 @@ click on **Sign up now** and create a new account.
 
 ## Create your first Instance
 
-1. We support <span id="requirements-4"></span>, but we strongly recommend
+1. First you have to set up your device and install the operating system.
+  We support <span id="requirements-4"></span>, but we strongly recommend
```

Markdownlint

Element: span
Inline HTML

(MD033, no-inline-html)